test/suite/test_assert01.py

```diff
@@ -1,114 +0,0 @@
-#!/usr/bin/env python
```

Comment:
@keithbostic - it looks from the view here as though test_assert01 through 06 are being deleted, not updated to only operate when not in diagnostic mode.

Reply:
@agorrod
> it looks from the view here as though test_assert01 through 06 are being deleted, not updated to only operate when not in diagnostic mode.

I rewrote those tests pretty much from scratch, so I moved them into a new file, `test_timestamp26.py`.

Reply:
Got it - thanks.